route.go

```diff
@@ -39,6 +39,7 @@ func NewBaseRoute(method, path string, handler any, e *Engine, options ...func(*
 		Operation:           openapi3.NewOperation(),
 		OpenAPI:             e.OpenAPI,
 		RequestContentTypes: e.requestContentTypes,
+		Engine:              e,
```

Comment:
Maybe we weren't clear. 

We don't want to pass the whole engine here. Our intention here was that `OpenAPIConfig` is now field on `OpenAPI`.

The `OpenAPI` struct should look like this now:

```golang
type OpenAPI struct {
	Config OpenAPIConfig

	description            *openapi3.T
	generator              *openapi3gen.Generator
	globalOpenAPIResponses []openAPIResponse
}
```

You'll need to update a few places cause of this change.

Reply:
Ups, sorry about that, I moved the config to the OpenAPI struct and changed all the references to the configuration.
Let me know if there is anything else :)